src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketsIds;
+

```

Comment:
You lost `User`. Add `private Long userId`

Reply:
But I don't know if it's a mistake. Maybe this implement has some logic, cos this is response